From fe5bc84edd6f3fcac28b9c332df3af7688821419 Mon Sep 17 00:00:00 2001
From: David Wu <david.wu@rock-chips.com>
Date: Tue, 14 Jan 2020 11:17:05 +0800
Subject: [PATCH] include: linux: soc: rockchip: Add rgmii delayline id

Add rgmii delayline id, which we can store and read delayline
of rgmii at vendor storage.

Change-Id: If97a76d706136bb43432688237980e9760b1b5f0
Signed-off-by: David Wu <david.wu@rock-chips.com>
---
 include/linux/soc/rockchip/rk_vendor_storage.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/soc/rockchip/rk_vendor_storage.h b/include/linux/soc/rockchip/rk_vendor_storage.h
index 4dfc4573e815..497d11157b85 100644
--- a/include/linux/soc/rockchip/rk_vendor_storage.h
+++ b/include/linux/soc/rockchip/rk_vendor_storage.h
@@ -25,6 +25,7 @@
 #define PLAYREADY_ROOT_KEY_1_ID		12
 #define SENSOR_CALIBRATION_ID		13
 #define IMEI_ID				15
+#define LAN_RGMII_DL_ID			16
 
 int rk_vendor_read(u32 id, void *pbuf, u32 size);
 int rk_vendor_write(u32 id, void *pbuf, u32 size);
-- 
2.35.3

